source/about-mongosync.txt

```diff
@@ -157,6 +157,73 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization.
+
+Consistency 
+~~~~~~~~~~~
+
+For the destination cluster to have an up to date, consistent view of the 
+source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+If the source cluster shuts down before ``mongosync`` can commit, such as in 
+a disaster scenario, the destination cluster might not have a consistent 
+snapshot of the source data, and transactions might not appear atomically on 
+the destination cluster. This can occur because ``mongosync`` temporarily alters 
```

Comment:
I think we may want to merge the behavior section into here (including the reordering/combination of writes and the note on transactions) and then link to this section on the Behavior page. It'd be great if we could have one place where we state that mongosync, on top of temporarily altering these collection characteristics, also does those other modifications that make the destination not be an accurate representation of the source at any given time.

Reply:
discussed in zoom + later comment in this thread that we'll merge this content in the mongosync Behavior page with the migration of the Continuous Sync section to that page.